src/qt/buynamespage.cpp

```diff
@@ -78,10 +101,46 @@ void BuyNamesPage::onRegisterNameAction()
     ui->registerNameButton->setDefault(true);
 }
 
+// Returns empty string if available, otherwise a description of why it is not
+// available.
+QString BuyNamesPage::name_available(const QString &name) const
+{
+    std::string strName = name.toStdString();
```

Comment:
Nit: This can be `const`, as well as some others below.

Reply:
Thanks for catching that; will fix.

Reply:
Fixed.  Also fixed the same error in a few other places in the GUI code, as a separate commit.

Reply:
Thanks.